tests/parser/functions/test_unsafe_math.py

```diff
@@ -23,8 +23,8 @@ def foo(x: {typ}, y: {typ}) -> {typ}:
 
     contract_2 = """
 @external
-def foo(x: {}) -> {}:
-    return unsafe_{}(x, {})
+def foo(x: {in_typ}) -> {out_typ}:
```

Comment:
nit: `in_typ == out_typ` always, so can just use `{typ}` in both positions
```suggestion
def foo(x: {typ}) -> {typ}:
```